src/iop/lens.cc

```diff
@@ -242,7 +242,7 @@ typedef struct dt_iop_lens_data_t
   float v_steepness;
   float reserved[2];
   float vigspline[VIGSPLINES];
-  uint32_t vighash;
+  dt_hash_t vighash;
```

Comment:
@TurboGit for the record, this is a minor bug but i will prepare a pr just for this and 4.6.1